components/pedigree/ui/src/main/resources/PhenoTips/PedigreeInterface.xml

```diff
@@ -47,13 +47,22 @@
 #end
 #macro(__getDisabledFields)
   #set ($recordConfiguration = $services.recordConfiguration.getActiveConfiguration())
-  #set ($disabled = ["first_name","last_name","last_name_birth"])
-  #foreach ($section in $recordConfiguration.enabledSections)
-    #foreach ($element in $section.enabledElements)
-      #if ($element == "Patient name")
-        #set ($disabled = [])
-        #break
-      #end
+  #set ($disabledFieldNames = [])
+  #set ($discard = $disabledFieldNames.addAll($recordConfiguration.allFieldNames))
+  #set ($discard = $disabledFieldNames.removeAll($recordConfiguration.enabledFieldNames))
+  #set ($linkedFields = {
+    "last_name": ["last_name_birth"],
+    "phenotype": ["hpo_positive"],
+    "negative_phenotype" : ["hpo_negative"],
+    "omim_id" : ["disorders"],
+    "gestation" : ["gestation_age"],
+    "gene" : ["candidate_genes", "causal_genes", "rejected_genes"]
+  })
+  #set ($disabled = [])
+  #set ($discard = $disabled.addAll($disabledFieldNames))
```

Comment:
as a side effect `disabled` will contain all disabled field names as configured in the patient form, e.g. `phenotype`, which are not the fields we want to disable in pedigree.

It does not hurt at the moment, but in theory it may overlap with a totally different field in pedigree (well, that is a bit stretching it, but still this is not a clean conversion). We should either rename pedigree fields, or have a clean conversion or - hopefully - get rid of pedigree node menu and use the same UI. Other than that, +1